lib/tasks/rubocop.rake

```diff
@@ -1,8 +1,21 @@
-if Rails.env.development? || Rails.env.test?
-  require 'rubocop/rake_task'
-
-  desc 'Execute rubocop -DR'
-  RuboCop::RakeTask.new(:rubocop) do |t|
-    t.options = ['-RDS'] # Rails, display cop name and styleguide link
+begin
+  require "rubocop"
+  require "rubocop/rake_task"
+rescue LoadError # rubocop:disable Lint/HandleExceptions
+else
+  Rake::Task[:rubocop].clear if Rake::Task.task_defined?(:rubocop)
+  desc 'Execute rubocop'
+  RuboCop::RakeTask.new(:rubocop) do |task|
+    task.formatters = ['progress']
+    task.options = ['--rails', '--display-cop-names', '--display-style-guide']
+    task.patterns = [
+      'Gemfile',
+      'Rakefile',
+      'lib/**/*.{rb,rake}',
+      'config/**/*.rb',
+      'app/**/*.rb',
+      'test/**/*.rb'
+    ]
```

Comment:
these patterns overrides whatever's in the `.rubocop.yml`. I've found it convenient to do it this way in the past, but don't feel strongly about it if you'd prefer to only have one definition. As I wrote in one of the commits, we could have less strict Rubocop behavior on CI, e.g. we could have

> bundle exec rake rubocop # won't run on test/
> bundle exec rubocop         # will run on test/